gpslogger/src/main/java/com/mendhak/gpslogger/loggers/customurl/CustomUrlJob.java

```diff
@@ -75,7 +78,35 @@ public Request authenticate(Route route, Response response) throws IOException {
 
         OkHttpClient client = okBuilder.build();
 
-        Request request = new Request.Builder().url(logUrl).build();
+        Request request;
+
+        if (usePost) {
+            FormBody.Builder bodyBuilder = new FormBody.Builder();
+            String[] urlSplit;
+            String[] paramSplit;
+            String baseUrl = "";
+            if (logUrl.contains("?")) {
+                urlSplit = logUrl.split("\\?");
+                if (urlSplit.length == 2) {
+                    baseUrl = urlSplit[0];
+                    paramSplit = urlSplit[1].split("\\&");
+                    for (int i = 0; i < paramSplit.length; i++) {
+                        if (paramSplit[i].contains("=")) {
+                            String[] oneParamSplit = paramSplit[i].split("=");
+                            if (oneParamSplit.length == 2) {
+                                bodyBuilder.add(URLEncoder.encode(oneParamSplit[0], "UTF-8"), URLEncoder.encode(oneParamSplit[1], "UTF-8"));
```

Comment:
Testing locally at the moment, I removed the `URLEncoder.encode` from here as it was causing double encoding.  Will push later.